### PR TITLE
Adding namespace UID as a label

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @rjnagal:

File: sinks/api/supported_labels.go [hunk 1]
```diff
@@ -53,6 +54,10 @@ var allLabels = []LabelDescriptor{
 		Description: "The namespace of the pod",
 	},
 	{
+		Key:         LabelPodNamespaceUID,
+		Description: "The UID of namespace of a pod",
```
Comment (a-robinson) [inline sinks/api/supported_labels.go]: Nit, but better phrased as "The UID of the namespace of the pod"

Comment (vishh) [inline sinks/api/supported_labels.go]: Done

**Comment by a-robinson** [comment]
LGTM

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**